sdk/core/azure-core/src/main/java/com/azure/core/util/IterableStream.java

```diff
@@ -1,39 +1,40 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-package com.azure.core.http.rest;
+package com.azure.core.util;
 
 import reactor.core.publisher.Flux;
 
 import java.util.Iterator;
 import java.util.stream.Stream;
 
 /**
- * This class provides utility to iterate over values. All the values are preserved even if they are traversed multiple times.
+ * This class provides utility to iterate over values using standard 'for-each' style loops, or to convert them into a
+ * Stream and operate in that fashion. All the values are preserved even if they are traversed multiple times.
  *
  * <p><strong>Code sample using Stream</strong></p>
  *
- * {@codesnippet com.azure.core.http.rest.iterableResponse.stream}
+ * {@codesnippet com.azure.core.util.iterableStream.stream}
```

Comment:
Glad you have codesnippets.. in the UX study, they had troubles with this class.